.prettierrc

```diff
@@ -1,16 +1,15 @@
 {
-  "jsxBracketSameLine": false,
```

Comment:
This changes alphabetizes this list and removes `jsxBracketSameLine` which is deprecated